code/mob/living/intangible/ai-camera.dm

```diff
@@ -453,6 +453,29 @@
 		if(mainframe)
 			mainframe.deploy_to()
 
+	verb/toggle_lock()
+		set category = "AI Commands"
+		set name = "Toggle Cover Lock"
+		if(mainframe)
```

Comment:
Why not just call the proc on the mainframe?

Reply:
.....very good question actually